go/service/gregor.go

```diff
@@ -1391,19 +1418,21 @@ type timeoutClient struct {
 var _ rpc.GenericClient = (*timeoutClient)(nil)
 
 func (t *timeoutClient) Call(ctx context.Context, method string, arg interface{}, res interface{}) error {
-	timeoutCtx, timeoutCancel := context.WithTimeout(ctx, t.timeout)
+	var timeoutCancel context.CancelFunc
+	ctx, timeoutCancel = context.WithTimeout(ctx, t.timeout)
```

Comment:
Yeah I just undid the change and committed that. I thought reassigning to `ctx` was more typical, but I'm happy to change it all back if you think that is better.